config.md

```diff
@@ -25,19 +25,61 @@ Changing it means creating a new derived image, instead of changing the existing
 
 ### Layer DiffID
 
-A layer DiffID is a SHA256 digest over the layer's uncompressed tar archive and serialized in the descriptor digest format, e.g., `sha256:a9561eb1b190625c9adb5a9513e72c4dedafc1cb2d4c5236c9a6957ec7dfd5a9`.
+A layer DiffID is the digest over the layer's uncompressed tar archive and serialized in the descriptor digest format, e.g., `sha256:a9561eb1b190625c9adb5a9513e72c4dedafc1cb2d4c5236c9a6957ec7dfd5a9`.
```

Comment:
> That is not a requirement.

With the old wording (“A layer DiffID is a SHA256 digest”) it certainly seems like using a `sha256` digest is a requirement.  Regardless of the original intent, clarifying this point seems orthogonal to documenting the chain ID.  But *shrug*, I'm not a maintainer.